tests/unit/test_missing_data.py

```
@@ -0,0 +1,215 @@
+        
+        assert len(result) == 2  # Only rows 0 and 3 are complete
+    
+    def test_returns_counts_when_requested(self):
+        """Test that counts are returned when return_counts=True."""
+        df = pd.DataFrame({
+            'a': [1, np.nan, 3, 4],
+            'b': [10, 20, np.nan, 40]
+        })
+        var_meta = {}
+        
+        result, counts = handle_missing_for_analysis(df, var_meta, return_counts=True)
```

Ruff (0.14.13)

Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)